try {
    let i = 10
    console.log(i / 0)
} catch (e) {
    console.log(e)
}